projects/igniteui-angular/src/lib/drop-down/drop-down-item.component.ts

```diff
@@ -15,6 +15,41 @@ import { IgxDropDownItemBase } from './drop-down-item.base';
     templateUrl: 'drop-down-item.component.html'
 })
 export class IgxDropDownItemComponent extends IgxDropDownItemBase implements DoCheck {
+    /**
+     * @inheritdoc
+     */
+    get focused(): boolean {
+        const focusedState = this._index !== null && this._index !== undefined
+        ? this._index === this.selection.first_item(`${this.dropDown.id}-active`) : this._focused;
+        return !this.isHeader && !this.disabled && focusedState;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    set focused(value: boolean) {
+        this._focused = value;
+    }
+    /**
+     * @inheritdoc
+     */
+    get selected(): boolean {
+        if (this._index) {
+            const item = this.dropDown.selectedItem;
```

Comment:
This will also skip the first index, but more importantly the `selectedItem` getter has a case where it'll check the selected property on the item before returning it, which is some cases (non-virtual, with bound index) will cause a Maximum call stack exception. This should not refer to `selectedItem` and the getter itself might not need to make those extra checks.